api/src/main/java/io/opentelemetry/trace/SpanId.java

```diff
@@ -80,8 +80,9 @@ public static SpanId getInvalid() {
    *
    * @param random The random number generator.
    * @return a valid new {@code SpanId}.
+   * @since 0.1.0
    */
-  static SpanId generateRandomId(Random random) {
+  public static SpanId generateRandomId(Random random) {
```

Comment:
We discussed about this method before, the conclusion back then was not to expose it in the API. See https://github.com/open-telemetry/opentelemetry-java/pull/382#discussion_r290976901.

Reply:
I have moved ID generation methods to SpanBuilder in SDK